DDD.Functions.Extensions/KeyDatesConfig.cs

```diff
@@ -50,6 +50,14 @@ public class KeyDatesConfig : Attribute
         public string StopSyncingTitoFrom { get; set; }
         public DateTimeOffset StopSyncingTitoFromDate => StopSyncingTitoFrom != null ? DateTimeOffset.Parse(StopSyncingTitoFrom) : DateTimeOffset.MinValue;
 
+        // Feedback
+        [AppSetting(Default = "FeedbackAvailableFrom")]
```

Comment:
These variables need to be set for the test and prod sites via Azure DevOps release variables and passed through in the ARM and PowerShell

Reply:
Can we use the "on the day" setting we already have?

Reply:
I believe "on the day" is not available in the backend side. Unfortunately It is only front end :(